ql/time/calendars/unitedstates.hpp

```diff
@@ -147,6 +159,7 @@ namespace QuantLib {
       public:
         //! US calendars
         enum Market { Settlement,     //!< generic settlement calendar
+                      LiborImpact,    //!< Libor impact calendar
```

Comment:
I would put this last, on the off-chance that someone was storing the numeric values.